### DISCUSSION TITLE
Google日曆的說明的圖示會變成文字[note]

### DISCUSSION THREAD

**Discussion by MrLuke1110** [F9y4ng/GreasyFork-Scripts#318] [问答专区(Question / Answer)] [answered]
Google日曆, 建立行程時，輸入說明的圖案，建立後會變成「note」，
如果不麻煩的話再協助處理，謝謝。

使用的瀏覽器是 Google Chrome
使用的字體是 等距更紗黑體TC
其他設定沒動

Google日曆 https://calendar.google.com/
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/66194793/9531a873-bc67-4347-abab-96100b733cc5)
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/66194793/f2e4e97c-3d2d-459a-8007-d0d51c627019)
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/66194793/b3b1f750-ae01-4467-8fc9-62e85bc282eb)

**Comment by F9y4ng** [comment, marked as answer]
在**需要渲染的网页元素**中，追加`:not(i *)`,保存为站点独享数据。

**Reply by MrLuke1110** [reply]
可以解決，感謝回覆
